Index.html

```diff
@@ -0,0 +1,441 @@
+      </li>
+      <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+      <li><a class="nav-link" href="#Variables">Variables</a></li>
+      <li>
+        <a class="nav-link" href="#Declaring_variables">Declaring variables</a>
+      </li>
+      <li><a class="nav-link" href="#Variable_scope">Variable scope</a></li>
+      <li><a class="nav-link" href="#Global_variables">Global variables</a></li>
+      <li><a class="nav-link" href="#Constants">Constants</a></li>
+      <li><a class="nav-link" href="#Data_types">Data types</a></li>
+      <li>
+        <a class="nav-link" href="#if...else_statement">if...else statement</a>
+      </li>
+      <li><a class="nav-link" href="#while_statement">while statement</a></li>
+      <li>
+        <a class="nav-link" href="#Function_declarations"
+          >Function declarations</a
+        >
+      </li>
+      <li><a class="nav-link" href="#Reference">Reference</a></li>
+    </ul>
+  </nav>
+  <main id="main-doc">
+    <section class="main-section" id="Introduction">
+      <header>Introduction</header>
+      <article>
+        <p>
+          JavaScript is a cross-platform, object-oriented scripting language. It
+          is a small and lightweight language. Inside a host environment (for
+          example, a web browser), JavaScript can be connected to the objects of
+          its environment to provide programmatic control over them.
+        </p>
+  
+        <p>
+          JavaScript contains a standard library of objects, such as Array, Date,
+          and Math, and a core set of language elements such as operators, control
+          structures, and statements. Core JavaScript can be extended for a
+          variety of purposes by supplementing it with additional objects; for
+          example:
+        </p>
+        <ul>
+          <li>
+            Client-side JavaScript extends the core language by supplying objects
+            to control a browser and its Document Object Model (DOM). For example,
+            client-side extensions allow an application to place elements on an
+            HTML form and respond to user events such as mouse clicks, form input,
+            and page navigation.
+          </li>
+          <li>
+            Server-side JavaScript extends the core language by supplying objects
+            relevant to running JavaScript on a server. For example, server-side
+            extensions allow an application to communicate with a database,
+            provide continuity of information from one invocation to another of
+            the application, or perform file manipulations on a server.
+          </li>
+        </ul>
+      </article>
+    </section>
+    <section class="main-section" id="What_you_should_already_know">
+      <header>What you should already know</header>
+      <article>
+        <p>This guide assumes you have the following basic background:</p>
+  
+        <ul>
```

Comment:
I think you better remove all blank line. 🔧

Reply:
- Please you can try to use polite and calm phrases to request for a change, so as the student can have courage to continue the work.